internal/server/proto/server.proto

```diff
@@ -2680,6 +2680,11 @@ message Release {
   // ID of the job that created this. This may be empty.
   string job_id = 13;
 
+  // A boolean to mark this release message as unimplemented by the plugin
+  // that generated the message. If true, that means a release was never
+  // executed and Waypoint failed to build a valid releaser.
+  bool unimplemented = 15;
```

Comment:
I also will nitpick this comment to be a little more clear on how Waypoint works: a `Release` is _always_ made, but this means that there was no actual logic run, this was just automatically created.

Reply:
Ah, good point! Important distinction to make.